app/assets/javascripts/discourse/tests/acceptance/sidebar-user-categories-section-test.js

```diff
@@ -491,3 +491,66 @@ acceptance("Sidebar - Logged on user - Categories Section", function (needs) {
     );
   });
 });
+
+acceptance(
+  "Sidebar - Logged on user - Categories Section - sidebar_destination_topic set to unread",
```

Comment:
I would avoid creating an entire acceptance module just for this case. Instead, you can use `updateCurrentUser` in a new test to alter the state of the user.